### DISCUSSION TITLE
Very slow sending rate

### DISCUSSION THREAD

**Discussion by martandya** [postalserver/postal#3222] [Help with using Postal]
Experience very slow sending using postal, approx 45K message are in pending status.
able to send only 30 Emails per min, Is there any way that we can increase it ?
Also, Ho can i increase containerized worker for postal ?

**Comment by willpower232** [comment]
I think you're essentially asking the same question twice here as workers are responsible for sending things.

You can scale up the workers to increase your sending rate, as long as you have CPU and memory to spare, have a look in the issues for `--scale`
